# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/webrtc-audio-processing-1/hotfix-ia64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- webrtc-audio-processing-v2.1/webrtc/rtc_base/system/arch.h.vanilla	2025-12-04 15:15:00.423533421 +0100
+++ webrtc-audio-processing-v2.1/webrtc/rtc_base/system/arch.h	2025-12-04 15:17:00.054714102 +0100
@@ -89,6 +89,9 @@
 #elif defined(__EMSCRIPTEN__)
 #define WEBRTC_ARCH_32_BITS
 #define WEBRTC_ARCH_LITTLE_ENDIAN
+#elif defined(__ia64)
+#define WEBRTC_ARCH_64_BITS
+#define WEBRTC_ARCH_LITTLE_ENDIAN
 #else
 #error Please add support for your architecture in rtc_base/system/arch.h
 #endif
